tests/Integration/Core/Module/ModuleRepositoryTest.php

```diff
@@ -68,4 +104,12 @@ public function testGetAtLeastOneModuleFromUniverse(): void
     {
         $this->assertGreaterThan(0, count($this->moduleRepository->getList()));
```

Comment:
Since the hook is now mocked with more info maybe you can test the `getList` a bit better? At least assert the data mocked for the hook are also part of the list?